com.chartboost.mediation/Android~/chartboost-mediation-android-bridge/src/main/java/com/chartboost/mediation/unity/BannerAdWrapper.kt

```diff
@@ -211,8 +209,8 @@ class BannerAdWrapper(private val ad: HeliumBannerAd) {
             else -> 0
         })
         json.put("aspectRatio", size?.aspectRatio)
-        json.put("width", creativeSize?.width ?: { size?.width })
-        json.put("height", creativeSize?.height ?: { size?.height })
+        json.put("width", creativeSize?.width ?: 0)
```

Comment:
if the creative size is null we just default to 0?? We will no longer fallback to the given size being requested?

Reply:
Requested size is already available through `bannerView.request.size` in the new API. It doesn't fallback to requested size anymore because Unity sets placement and size when it makes a load request. if `bannerView.getSize()` is called before load request then it would return a default size which could be different on Android and IOS. It is set to  0,0 since that's how Unity treats it as a default case